examples/experimental/json-browser/app.js

```diff
@@ -215,6 +215,10 @@ export default class App extends Component {
                 id="json-deck"
                 {...jsonProps}
                 viewState={viewState}
+                pickingRadius={5}
+                getTooltip={object => {
+                  return {'Num points': object.elevationValue};
```

Comment:
This is only going to work with the `HexagonLayer`. Put something more generic here?

Reply:
I think I'll remove the example here in favor of putting the tooltips in our other examples

Reply:
Unless you think it makes sense to roll into this PR, I'll address the examples in a separate one